chains/blast/logo.svg

```diff
@@ -1 +1,5 @@
-<svg xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="14.6 46.1 185.8 122.8"><path fill="#FCFC03" d="m161 106.4 29.3-14.6 10.1-31-20.2-14.7H45.7L14.6 69.2h158.1l-8.4 26h-63.4l-6.1 19h63.4l-17.8 54.7 29.7-14.7 10.6-32.8-19.9-14.6.2-.4Z"/><path fill="#FCFC03" d="m59.3 145.4 18.3-57-20.3-15.2-30.5 95.7h113.6l7.6-23.5H59.3Z"/></svg>
\ No newline at end of file
+<svg version="1.0" xmlns="http://www.w3.org/2000/svg" width="300.000000pt" height="300.000000pt" viewBox="50 50 200 200" preserveAspectRatio="xMidYMid meet">
```

Comment:
@avious00 Did you mean to add explicit width and heights here? I think this will force extra space around the svg.

Reply:
removed